performance.md

```diff
@@ -222,4 +222,4 @@ And that we would call from the cron with the following query:
 */5 * * * /usr/bin/php /home/example.com/run-wp-cron.php > /dev/null 2>&1
 ```
 
-There are many ways to make the calls corresponding to the scheduled tasks, so it is highly recommended that you talk to your web hosting provider for recommendations.
\ No newline at end of file
+There are many ways to make the calls corresponding to the scheduled tasks, so it is highly recommended that you talk to your web hosting provider for recommendations.
```

Comment:
The same here -- let's recommend or give options for what is recommended, rather than suggesting a host contact their host.